app/src/main/java/com/iyehuda/feelslike/ui/home/HomeFragment.kt

```diff
@@ -4,25 +4,48 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.fragment.app.viewModels
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.iyehuda.feelslike.R
 import com.iyehuda.feelslike.databinding.FragmentHomeBinding
-import com.iyehuda.feelslike.ui.base.BaseFragment
-import dagger.hilt.android.AndroidEntryPoint
 
-@AndroidEntryPoint
-class HomeFragment : BaseFragment<FragmentHomeBinding>() {
-    private val viewModel: HomeViewModel by viewModels()
+class HomeFragment : Fragment() {
```

Comment:
Inherit from `BaseFragment` (see previous code)
That way `binding` definition, `onDestroyView` and `onCreateView` code are replaced by short `createBinding` function